TestApps/SIL.Windows.Forms.TestApp/TestAppForm.cs

```diff
@@ -490,5 +491,23 @@ private void btnMediaFileInfo_Click(object sender, EventArgs e)
 				}
 			}
 		}
+
+		private void btnShowFileOverwriteDlg_Click(object sender, EventArgs e)
+		{
+			var filenames = new List<string>
+			{
+				@"c:\folder\file.txt",
+				@"My Documents\another.doc",
+				@"LastOne.png"
+			};
+			foreach (var file in filenames)
+			{
+				using (var dlg = new ConfirmFileOverwriteDlg(file))
+				{
+					if (dlg.ShowDialog(this) == DialogResult.No && dlg.ApplyToAll)
```

Comment:
Can "yes" not also be applied to all?
This is not the behaviour I'd expect, but I suppose it's ok for a test app.

Reply:
Apply to All can (of course) be interpreted however the client wishes to interpret it. But, yes, my normal expectation is that it would indeed apply to all. Since in this test app, we're not dealing with real files and not actually overwriting them, I just did this as a mindless way to demonstrate that both the dialog result and the value of ApplyToAll are being set as expected. I'll try to make it a little better.

Reply:
Much nicer!